core/src/test/scala/com/raphtory/stateTest/AllCommandsTest.scala

```diff
@@ -16,61 +16,61 @@ import scala.language.postfixOps
 // Broken until we have a proper JSON spout
 @DoNotDiscover
 class AllCommandsTest extends BaseRaphtoryAlgoTest[String] {
-  test("Graph State Test") {
-    val sink = FileSink(outputDirectory)
-
-    val results = algorithmTest(
-            algorithm = GraphState(),
-            sink = sink,
-            start = 1,
-            end = 290001,
-            increment = 10000,
-            windows = List(1000, 10000, 100000, 1000000)
-    )
-
-    logger.info(s"Graph state test finished. Result is '$results'.")
-
-    val expected = "a52efc4b1962f961503ad3fabf3893e3a384056c3ce52342fa43fbdec80c08a0"
-
-    assert(results equals expected)
-  }
-
-  test("Connected Components Test") {
-    val sink = FileSink(outputDirectory)
-
-    val results = algorithmTest(
-            algorithm = ConnectedComponents(),
-            sink = sink,
-            start = 1,
-            end = 290001,
-            increment = 10000,
-            windows = List(1000, 10000, 100000, 1000000)
-    )
-
-    logger.info(s"Connected components test finished. Result is '$results'.")
-
-    val expected = "ba662e8fe795c263566b1898c0f1fc6816c3f6ef5e6bbbce9db6cd06330f47a8"
-
-    assert(results equals "ba662e8fe795c263566b1898c0f1fc6816c3f6ef5e6bbbce9db6cd06330f47a8")
-  }
-
+//  test("Graph State Test") {
```

Comment:
This can probably be fixed now, but does it need to be commented if it is already DoNotDiscover

Reply:
DoNotDiscover doesn't apply to `munit`